### ISSUE TITLE
Menderbug's house doesn't open

### ISSUE THREAD

**Issue by IdkDiegoiguess** [Korzer420/LoreMaster#18]
I tried getting Menderbug's Lore for completion's sake but after killing Menderbug (also known as slashing until the dream text said "Don't say I didn't warn you") the house doesn't open, I am not sure if it's something like Mr. Mushroom where you have to have all 3 dreamers or something but I can't find anything saying that.

**Comment by IdkDiegoiguess** [comment]
Enabled Mods:
AdditionalChallenge
Additional Maps
BenchWarp
BenchWarpFix
Better_Void_Heart
CharmChanger
CharmPreset
CMICore
CustomKnight
Damage Values (for some reason doesn't work)
DebugMod
EnemyHPBar
Fast Travel
FStatsMod
Godseeker Memory Jump
GodseekerPlus
Hall of Gods Extras
HKMirrorMod
HKTool
ItemChangerMod
LoreMaster
LostArtifacts
MagicUI
MapChangerMod
MenuChanger
MoreSaves
NoSpidersMod
Osmi
QoL
RecentItems
Satchel
SFCore
SkillUpgrades
SpeedrunQoL
Toggle_Rando_Split_Options Bindings
ToggleableBindings
Transcendence
VanillaMapMod
Zote Killer Mod

**Comment by Korzer420** [comment]
Hm, ItemChanger should automatically open Menderbugs house. Maybe it got changed in the setting of ItemChanger. Go under Options > Mod > ItemChanger and look if you can find something. If it got changed though, I'll re-add that manually in the next update

**Comment by xKurisuX** [comment]
I can't open the menderbug home either and it's bugging me. i did the same thing, attack menderbug until his voice line changed then I went and chencked, and it never opened.

**Comment by IdkDiegoiguess** [comment]
Is there a fix anyone has found?

**Comment by Korzer420** [comment]
If ItemChanger did change this module, this will be readded in the next lore master update. Since I currently don't have the time nor motivation to work on this now, this might take a few weeks. 

If you want the power anyway you have three options:
- Play the extra mode and add the missing ability via a scroll of knowledge.
- Close your game, open your userX.modded.json file and add "MENDERBUG" in the "ObtainedPowers" field.
It then should look like this:
```"ObtainedPowers":
[
    "MENDERBUG"
]
```
- Create an optionsX.txt file in the directory of the mod and paste this in
```
%Modify%
%BestMenderInTheWorld% global | add
```
and reenter you save file

**Comment by IdkDiegoiguess** [comment]
Thanks

**Comment by Korzer420** [comment]
Should now be fixed with 2.3.3
